index.html

```diff
@@ -6802,6 +6802,32 @@ <h3>Fitness for Purpose</h3>
 [=verifiers=] unless they accept the liability of ignoring the policy.
         </p>
       </section>
+
+      <section class="informative">
+        <h3>Artificial Intelligence</h3>
+
+        <p>
+Systems using artificial intelligence are advancing to the point where they can
+perform a variety of complex tasks to a level that increasingly often meets or
+exceeds human performance. As a result, such advanced
+artificial-intelligence-based systems have an increased ability to acquire and
+use [=verifiable credentials=] in ways that might have traditionally signaled a
+human intelligence; that signal might no longer be accurate.
```

Comment:
The language here seemed complex to me, I tried to simplify it:

```suggestion
Systems using artificial intelligence might be capable of performing
complex tasks at a level that meets or exceeds human performance.
This might include tasks such as the acquisition and use of
[=verifiable credentials=]. Therefore, using such tasks as a signal of
human activity might not be accurate.
```